components/common-go/kubernetes/kubernetes.go

```diff
@@ -64,6 +64,12 @@ const (
 
 	// WorkspaceSSHPublicKeys contains all authorized ssh public keys that can be connected to the workspace
 	WorkspaceSSHPublicKeys = "gitpod.io/sshPublicKeys"
+
+	// workspaceCpuLimit denotes the cpu limit of a workspace
+	WorkspaceCpuMinLimitAnnotation = "gitpod.io/cpuMinLimit"
```

Comment:
why annotation is called `cpuMinLimit`? It implies this is a minimum limit set on workspace? :thinking: 
Maybe you could clarify that in the comment above?